apps/src/blockly/addons/contextMenu.ts

```diff
@@ -440,10 +440,59 @@ function registerHelp(weight: number) {
   GoogleBlockly.ContextMenuRegistry.registry.register(helpOption);
 }
 
+export enum WeightOptions {
+  CUT = 0,
+  COPY = 1,
+  PASTE = 2,
+  DELETE = 3,
+  DUPLICATE = 4,
+  DISABLE = 5,
+  MOVE = 6,
+  EDIT = 7,
+  MOVE_COMMENT = 8,
+  UNDO = 9,
+  REDO = 10,
+  COMMENT = 11,
+  HELP = 12,
+  CLEANUP = 13,
+  LEVELBUILDER = 14,
+}
+
 export const registerAllContextMenuItems = function () {
   unregisterDefaultOptions();
-  registerCustomBlockOptions();
-  registerCustomWorkspaceOptions();
+
+  const menuWeightMap: Record<string, WeightOptions> = {
+    blockCopyToStorage: WeightOptions.COPY,
+    blockPasteFromStorage: WeightOptions.PASTE,
+    blockDelete: WeightOptions.DELETE,
+    blockDuplicate: WeightOptions.DUPLICATE,
+    blockDisable: WeightOptions.DISABLE,
+    undoWorkspace: WeightOptions.UNDO,
+    redoWorkspace: WeightOptions.REDO,
+    blockComment: WeightOptions.COMMENT,
+    cleanWorkspace: WeightOptions.CLEANUP,
+  };
+  for (const [option, weight] of Object.entries(menuWeightMap)) {
+    overrideOptionWeight(option, weight);
+  }
+  registerHelp(WeightOptions.HELP); // Custom student-facing help option.
+
+  // Our levelbuilder options should all show below the registered default options.
+  let nextWeight = WeightOptions.LEVELBUILDER;
```

Comment:
philosophical nit: since do a post-increment when passing this value into each call to `registerDeletable`, is it actually `currentWeight`?